composer.json

```diff
@@ -17,9 +17,13 @@
     "require": {
         "php": "^7.2 || ^8.0",
         "ext-json": "*",
-        "guzzlehttp/guzzle": "^7.0"
+        "php-http/discovery": "^1.13",
+        "php-http/multipart-stream-builder": "^1.1",
+        "psr/http-factory": "^1.0"
     },
     "require-dev": {
+        "guzzlehttp/guzzle": "^7.0",
```

Comment:
Brauchts des dann überhaupt noch?

Reply:
Yep, to test the auto discovery of the client